medical/models/medical_patient.py

```diff
@@ -101,7 +101,7 @@ def _create_vals(self, vals):
         })
         return vals
 
-    @api.model
+    @api.model_cr_context
     def _get_default_image(self, vals):
         res = super(MedicalPatient, self)._get_default_image(vals)
```

Comment:
Yup exactly. Sorry, that's what I meant by the inheritance chain. More the chain of the creates, and the order in which they always happen.

Are you suggesting we make this more explicit? The only time I was able to make the partner default show instead of the patient was to create the partner as a standard type, then escalate to a medical entity. In that case, the partner image would apply - but that's mainly just because it already has one, so the default doesn't trigger.